app/tandem/Context.h

```diff
@@ -48,12 +49,16 @@ template <typename Type> class Context : public ContextBase {
 public:
     using adapter_t = AdapterOperator<Type>;
     using dg_t = DGOperator<Type>;
-    using friction_lop_t = RateAndState<DieterichRuinaAgeing>;
+    #if defined(DR_AGING_LAW)
+        using friction_lop_t = RateAndState<DieterichRuinaAging>;
```

Comment:
best would be to avoid compile time choices, but to read it from the toml file.
(but I suggest you first fix all my other comments, and then we can discuss how to implement that).

Reply:
Ok the run-time was a bit overambitious (would probably require modifying Context to add a template frictionlawtype template, but I think it is not worth implementing at this stage).